docs/src/languages/php/extensions.md

```diff
@@ -60,110 +60,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| mailparse        |     |     |     | *   | *   | *   |     | *   |     |
+| mbstring         |     |     |     | *   | *   | *   | *   | *   | *   |
+| mcrypt           | *   | *   | *   | *   | *   |     |     |     |     |
+| memcache         | *   | *   | *   |     |     |     |     |     |     |
+| memcached        | *   | *   | *   | *   | *   | *   |     | *   |     |
+| mongo            | *   | *   | *   |     |     |     |     |     |     |
+| mongodb          |     |     |     | *   | *   | *   | *   | *   |     |
+| msgpack          |     |     | *   | *   | *   | *   |     | *   |     |
+| mssql            | *   | *   | *   |     |     |     |     |     |     |
+| mysql            | *   | *   | *   |     |     |     |     |     | *   |
+| mysqli           | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| mysqlnd          | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| newrelic         |     |     | *   | *   | *   | *   | *   | *   |     |
+| oauth            |     |     |     | *   | *   | *   | *   | *   |     |
+| odbc             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| opcache          |     | *   | *   | *   | *   | *   | *   | *   | *   |
+| openssl          |     |     |     |     |     |     |     |     |     |
+| pdo              | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_dblib        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_firebird     | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_mysql        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_odbc         | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_pgsql        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_sqlite       | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_sqlsrv       |     |     |     | *   | *   | *   |     | *   |     |
+| http             |     |     | *   |     |     |     |     | *   |     |
+| pgsql            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| phar             |     |     |     | *   | *   | *   | *   | *   |     |
+| pinba            | *   | *   | *   |     |     |     |     |     |     |
+| posix            |     |     |     | *   | *   | *   | *   | *   |     |
+| propro           |     |     | *   |     |     |     |     |     |     |
+| pspell           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pthreads         |     |     |     |     | *   |     |     |     |     |
+| raphf            |     |     | *   |     |     |     |     | *   | *   |
+| readline         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| recode           | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| redis            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| shmop            |     |     |     | *   | *   | *   | *   | *   |     |
+| simplexml        |     |     |     | *   | *   | *   | *   | *   |     |
```

Comment:
Also a bundled extension we should include.

Reply:
I've missed all the buildin extensions thanks fixed in 28e56edf24e07dc8fc1e1dc573bd5c38d418b61a